config/admin_ranks.txt

```diff
@@ -47,3 +47,13 @@ Host			= +EVERYTHING *EVERYTHING
 
 Head Coder      = +EVERYTHING
 Coder			= +DEBUG +VAREDIT +SERVER +SPAWN +POLL -AUTOLOGIN
+
+
+Aspirant 	= +ADMIN +BAN +REJUV
+Knight 		= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +SOUND +SPAWN
+Paladin		= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +BUILD +SOUND +SPAWN +SERVER +RIGHTS +DJ +POLL +VAREDIT
+Sentinel	= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +BUILD +SOUND +SPAWN +SERVER +RIGHTS +DJ +POLL +VAREDIT
+Scribe 		= +STEALTH +POSSESS +REJUV +BUILD +DEBUG +VAREDIT +SOUND +SPAWN +DJ +POLL
+Head Scribe = +FUN +STEALTH +POSSESS +REJUV +BUILD +SERVER +DEBUG +VAREDIT +SOUND +SPAWN +RIGHTS +DJ +POLL
```

Comment:
They don't need +Rights, or +Fun

Reply:
Neither do they need +DJ or +Sound

Reply:
sentinels (head admininstrator) needs to be able to promote in game. paladins double as admin trainers and need to be able to promote people to aspirant.